Extensions/xkit_preferences.js

```diff
@@ -1680,8 +1680,8 @@ XKit.extensions.xkit_preferences = new Object({
 
 			var has_css = m_extension.css !== "";
 			var has_icon = m_extension.icon !== "";
-			var is_beta = m_extension.beta === true;
-			var is_frame = m_extension.frame === true;
+			var is_beta = m_extension.beta;
+			var is_frame = m_extension.frame;
```

Comment:
Upon closer inspection, I'm not sure if this block gets run at all. It's supposed to get called when the element with the id `#xkit-extension-more-info` gets clicked, but that has a default style of `display: none;`, and I don't see any code that removes that style.